CODEOWNERS

```diff
@@ -327,7 +327,22 @@ amd64/                                      @tkreuzer
 # Translations
 # This is the list of translation teams in ReactOS GitHub organization.
 # If you want to be part of one - hit us at https://chat.reactos.org/
-de-DE.rc    @reactos/lang-german
-fr-FR.rc    @reactos/lang-french
-nl-NL.rc    @reactos/lang-dutch
-ru-RU.rc    @reactos/lang-russian
+de-DE.*    @reactos/lang-german
```

Comment:
Why .* instead of .rc ?

Reply:
See the explanation at the top of the PR.

Reply:
Right.

Reply:
Would there be a usable pattern/mean to explicitly list `h` and `rc` only?

Reply:
There are also `.mc` and files without extension.